src/features/donation/models/effects/campaign-ft-donation.ts

```
@@ -0,0 +1,243 @@
+          FungibleTokenStorageBalance | null
+        >("storage_balance_of", { args: { account_id: referrerAccountId } })
+        .then((response) =>
+          indivisibleUnitsToBigNum(response?.total ?? String(0), NATIVE_TOKEN_DECIMALS),
+        )
+    : null;
+
+  return Promise.all([
+    tokenClient.view<{}, FungibleTokenMetadata>("ft_metadata"),
+
+    tokenClient
+      .view<{}, { min: string; max: string }>("storage_balance_bounds")
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)